examples/volumes/glusterfs/README.md

```diff
@@ -103,6 +104,7 @@ The following *volume* spec in [glusterfs-pod.json](glusterfs-pod.json) illustra
 The parameters are explained as the followings.
 
 - **endpoints** is endpoints name that represents a Gluster cluster configuration. *kubelet* is optimized to avoid mount storm, it will randomly pick one from the endpoints to mount. If this host is unresponsive, the next Gluster host in the endpoints is automatically selected.
+- **endpointsns** is the namespace name for the above mentioned endpoint.
```

Comment:
- add `optional`, default is the pod's namespace (backward compatible)
- use full name `endpointNamespace`.